Updated readme and action file (#3)
go-feature-flag/gofeatureflag-lint-action: README.md

<p align="center">
    <img width="250" height="238" alt="Go Feature Flag" src="https://github.com/thomaspoignant/go-feature-flag/raw/main/logo.png" />
    <h2 align="center">GoFeatureFlagLint</h2>
    <p align="center">Lint your feature flag config files</p>
</p>

The `go-feature-flag/gofeatureflag-lint-action` Github action performs the

> linting of the [gofeatureflag](https://gofeatureflag.org/) flag config file. It

Reviewer: ```suggestion
linting of the [GO Feature Flag](https://gofeatureflag.org/) flag config file. It
```

    <p align="center">Lint your feature flag config files.</p>
</p>

The `go-feature-flag/gofeatureflag-lint-action` Github action performs the
linting of the [gofeatureflag](https://gofeatureflag.org/) flag config file. It
checks for errors in the config file and returns the error message if any.

📢 Join Our Community: [![Slack](https://img.shields.io/badge/join-us%20on%20slack-gray.svg?longCache=true&logo=slack&colorB=green)](https://gophers.slack.com/messages/go-feature-flag)

## Usage 📝

```yaml
on: [push]

jobs:
  gofeatureflag-lint-job:
    runs-on: ubuntu-latest
    name: A demo job to run gofeatureflag-lint on a config file
    steps:
      
      ## checkout the code from the repository
      - name: Checkout
        uses: actions/checkout@v3
      
      ## run the gofeatureflag-lint action
      - name: Lint the config file

>         uses: go-feature-flag/gofeatureflag-lint-action@ea0524d562003105471e2ccdf3485e98db23fa48

Reviewer: I will suggest in the doc to add `v1` to prepare for the release when the tag will be ready.

```suggestion
        uses: go-feature-flag/gofeatureflag-lint-action@v1
```

Author: Yes, I agree and have updated the example.

        ## set the input parameters for the action
        with:

>           flagFilePath: ${{ github.workspace }}/config/flag-config.yaml
>           flagFiletype: 'yaml'

Reviewer: All examples of Github Action I see around are using [kebab case](https://www.theserverside.com/definition/Kebab-case) for parameters.

I will change this into something like this:
```suggestion
          flag-file: ${{ github.workspace }}/config/flag-config.yaml
          format: yaml
```

Author: I was using the kebab-case style, then later I tried to follow more Go style. I will update all the occurrence to kebab-case.

```

## Inputs ⚙️

| **Name** | **Required** | **Defaults**  | **Description**  |
|:-----:|:-----:|:-----:|:-----:|

> |  flagFilePath     | false      | `config/flag-config.yaml`  | Absolute or relative path of the file with name  |
> |   flagFiletype    | false      | `yaml`  | config file of type `yaml`, `toml`, or `json`  |

Reviewer: ```suggestion
|  `flag-file`     | false      | `config/flag-config.yaml`  | Absolute or relative path of the file with name  |
|   `format`    | false      | `yaml`  | config file of type `yaml`, `toml`, or `json`  |
```

Author: Done

## Outputs ✉️

| **Name** | **Description**  |
|:-----:|:-----:|

> |  lintMessage     | `empty` if no error is found, else the value will be the error message  |

Reviewer: ```suggestion
|  lint-message     | `empty` if no error is found, else the value will be the error message  |
```

Author: Done

## Share the Love ❤️

> Like this project? Please give it a ★ on our GitHub!

Reviewer: Love this ❤️

Author: Thanks 🙏
